tools/osbuild-depsolve-dnf5

```diff
@@ -101,9 +101,9 @@ class Solver():
 
         # Gather up all the exclude packages from all the transactions
         exclude_pkgs = []
-        transactions = arguments.get("transactions", [])
+        transactions = arguments.get("transactions") or []
```

Comment:
This is a very nuanced difference and I'd *really* like a comment here on the behavior, something like "The default only gets returned when the key does not exist. We need to return an empty list even if the value in `arguments` was `None`" perhaps with this snippet:

```
>>> D = {"foo": None}
>>> D.get("foo", [])
>>> D.get("bar", [])
[]
```